examples/MomentoUsage/Program.cs

```diff
@@ -2,22 +2,33 @@
 using Momento.Sdk.Config;
 using Momento.Sdk.Responses;
 
-String MOMENTO_AUTH_TOKEN = Environment.GetEnvironmentVariable("MOMENTO_AUTH_TOKEN");
-String CACHE_NAME = "cache";
-String KEY = "MyKey";
-String VALUE = "MyData";
-uint DEFAULT_TTL_SECONDS = 60;
+string MOMENTO_AUTH_TOKEN = Environment.GetEnvironmentVariable("MOMENTO_AUTH_TOKEN") ?? "";
```

Comment:
Do we want a default value here, or to throw an exception right away?

Reply:
I was planning on letting our improved error handling for auth tokens handle this.

Reply:
i think we should do both.  doesn't have to be in this PR but we should have an error that says "you forgot to set your env var", and another one that says "you passed a null or empty auth token".